apache-rat-plugin/src/main/java/org/apache/rat/mp/AbstractRatMojo.java

```diff
@@ -392,7 +392,7 @@ private void setIncludes(DirectoryScanner ds) throws MojoExecutionException {
                 }
                 includeList.addAll(getPatternsFromFile(f, charset));
             }
-            ds.setIncludes(includeList.toArray(new String[includeList.size()]));
+            ds.setIncludes(includeList.toArray(new String[0]));
```

Comment:
Following https://docs.oracle.com/javase/8/docs/api/java/util/Collection.html#toArray-T:A- setting the array's size prevents creating a new one.